packages/db-states-file/lib/states/statesInMemServerRedis.js

```diff
@@ -243,6 +252,13 @@ class StatesInMemoryServer extends StatesInMemoryFileDB {
                 } else {
                     handler.sendBulk(responseId, JSON.stringify(result));
                 }
+            } else if (namespace === this.metaNamespace) {
+                const result = this.getMeta(id);
+                if (!result) {
```

Comment:
is ! enought here because in fact we store values ... so it could also be 0 ...... or false

Reply:
if we decide that integeres or bools will be written there which we need to convert to strings first yes, else it is enough.